src/bitcoin-cli.cpp

```diff
@@ -88,10 +92,10 @@ static int AppInitRPC(int argc, char* argv[])
     //
     ParseParameters(argc, argv);
     if (argc<2 || mapArgs.count("-?") || mapArgs.count("-h") || mapArgs.count("-help") || mapArgs.count("-version")) {
-        std::string strUsage = _("Zcash RPC client version") + " " + FormatFullVersion() + "\n" + PrivacyInfo();
+        std::string strUsage = strprintf(_("%s RPC client version"), _(PACKAGE_NAME)) + " " + FormatFullVersion() + "\n" + PrivacyInfo();
         if (!mapArgs.count("-version")) {
             strUsage += "\n" + _("Usage:") + "\n" +
-                  "  zcash-cli [options] <command> [params]  " + _("Send command to Zcash") + "\n" +
+                  "  zcash-cli [options] <command> [params]  " + strprintf(_("Send command to %s"), _(PACKAGE_NAME)) + "\n" +
```

Comment:
Shouldn't this actually be "Send command to zcashd"?

Reply:
In upstream this would say "Send command to Bitcoin Core", because "Bitcoin Core" is the "package name" (as opposed to other distributions). We've aliased "Zcash" to mean several things here, and `zcashd` isn't necessarily the "package name" we want. So the question is whether here we want this to mean the package name for the server, or the binary name for the server.

Reply:
"Send command to Zcash" is meaningless to me. Zcash is a protocol and a block chain; it's not something you can send a command to. Also "send command" is ambiguous; it could mean executing `zcashd`. You have to already know what it is intended to mean in order for it to make sense.

Reply:
Given all your subsequent comments, it sounds like what you really want is to set `PACKAGE_NAME` to "zcashd". Shall we just do what?

Reply:
I don't know whether `PACKAGE_NAME` is used in executables other than `zcashd`. If it isn't, then yes this sounds fine.

Reply:
I checked all of the pre-existing uses:
```c++
./src/consensus/params.cpp:188:                throw std::runtime_error("Funding stream address was not a valid " PACKAGE_NAME " address.");
./src/wallet/walletdb.cpp:1346:    std::pair<std::string, std::string> networkInfo(PACKAGE_NAME, networkId);
./src/wallet/wallet.cpp:4233:    std::pair<string, string> networkInfo(PACKAGE_NAME, networkIdString);
./src/wallet/wallet.cpp:6603:                               walletFile, _(PACKAGE_NAME)));
./src/wallet/wallet.cpp:6606:            return UIError(strprintf(_("Wallet needed to be rewritten: restart %s to complete"), _(PACKAGE_NAME)));
./src/wallet/wallet.cpp:6610:            return UIError(strprintf(_("Wallet %s is not for %s %s network"), walletFile, _(PACKAGE_NAME), params.NetworkIDString()));
./src/wallet/rpcwallet.cpp:421:        throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, std::string("Invalid " PACKAGE_NAME " transparent address: ") + destStr);
./src/wallet/rpcwallet.cpp:1052:        throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, std::string("Invalid " PACKAGE_NAME " transparent address: ") + strAddress);
./src/wallet/rpcwallet.cpp:1112:        throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, std::string("Invalid " PACKAGE_NAME " transparent address: ") + destStr);
./src/wallet/rpcwallet.cpp:1287:            throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, std::string("Invalid " PACKAGE_NAME " transparent address: ") + name_);
./src/init.cpp:1910:                return InitError(_("-mineraddress is not a valid " PACKAGE_NAME " address."));
```
The one at `src/wallet/wallet.cpp:6606` could be "zcashd", but all of the rest should stay as "Zcash". So I think the best approach would be to add an `EXECUTABLE_NAME` or `DAEMON_NAME` variable and change the relevant uses of `PACKAGE_NAME` to that. But that does not need to be done in this PR.

Reply:
Done in #6814 (draft).

Reply:
I think most of those are misusing `PACKAGE_NAME` and actually mean "Zcash the network", not "Zcash the package maintained and distributed by ECC".